bpf/Makefile

```diff
@@ -267,7 +267,7 @@ bpf_lxc.o: bpf_lxc.ll
 	$(QUIET) ${LLC} ${LLC_FLAGS} -filetype=obj -o $@ $(patsubst %.o,%.ll,$@)
 
 CT_TEST_OPTIONS = -DENABLE_IPV4 -DENABLE_IPV6 -DENABLE_HOST_REDIRECT -DENABLE_NAT46 \
-	-DENABLE_ROUTING -DENABLE_IPSEC -DPOLICY_VERDICT_NOTIFY
+	-DENABLE_ENDPOINT_ROUTES -DENABLE_IPSEC -DPOLICY_VERDICT_NOTIFY
```

Comment:
and again

Reply:
will take care with -DENABLE_ENDPOINT_ROUTES=0